Refit.Tests/RequestBuilder.cs

```diff
@@ -1070,6 +1079,55 @@ public void QueryStringWithArrayCanBeFormatted()
             Assert.Equal("/query?numbers=1%2C2%2C3", uri.PathAndQuery);
         }
 
+        [Fact]
+        public void QueryStringWithArrayFormattedAsMulti()
```

Comment:
This and the following two tests could be rewritten as a `[Theory]`, which would make it simpler for you to test the remaining option. 

We're still missing tests for pipe and tab separated values.

Reply:
fixed